packages/playground/cli/src/mount.ts

```diff
@@ -0,0 +1,78 @@
+import { existsSync } from 'fs';
+import path from 'path';
+import { createNodeFsMountHandler } from '@php-wasm/node';
+import type { PHP } from '@php-wasm/universal';
+
+export interface Mount {
+	hostPath: string;
+	vfsPath: string;
+}
+
+/**
+ * Parse an array of mount argument strings where the host path and VFS path
+ * are separated by a colon.
+ * e.g. [ '/host/path:/vfs/path', '/host/path:/vfs/path' ]
```

Comment:
`````suggestion
 * Example:
 * ```ts
 * parseMountWithDelimiterArguments( [ '/host/path:/vfs/path', '/host/path:/vfs/path' ] )
 * // [ { hostPath: '/host/path', vfsPath: '/vfs/path' }, { hostPath: '/host/path', vfsPath: '/vfs/path' } ]
`````

Reply:
Thanks!